tests/wrappers/test_passive_env_checker.py

```diff
@@ -0,0 +1,100 @@
+import re
+
+import numpy as np
+import pytest
+
+import gym
+from gym.wrappers.env_checker import PassiveEnvChecker
+from tests.envs.test_envs import PASSIVE_CHECK_IGNORE_WARNING
+from tests.envs.utils import all_testing_initialised_envs
+from tests.testing_env import GenericTestEnv
+
+
+@pytest.mark.parametrize(
+    "env",
+    all_testing_initialised_envs,
+    ids=[env.spec.id for env in all_testing_initialised_envs],
+)
+def test_passive_checker_wrapper_warnings(env):
+    with pytest.warns(None) as warnings:
+        checker_env = PassiveEnvChecker(env)
+        checker_env.reset()
+        checker_env.step(checker_env.action_space.sample())
+        # todo, add check for render, bugged due to mujoco v2/3 and v4 envs
+
+        checker_env.close()
+
+    for warning in warnings.list:
+        if warning.message.args[0] not in PASSIVE_CHECK_IGNORE_WARNING:
+            raise gym.error.Error(f"Unexpected warning: {warning.message}")
```

Comment:
Do we also want to do `checker_env.render()` for some render mode?

Reply:
The issue I found was with `rgb_array` for mujoco. 
Due to the structure of `mujoco_envs.py`, it is difficult to work out what the cause of the issue is. 
Im planning on adding that once some class changes and the bug is fixed.